draft-ietf-ppm-dap.md

```diff
@@ -451,25 +454,28 @@ Measurement:
   {{!VDAF}}.
 
 Minimum batch size:
-: The minimum number of reports in a batch.
+: The minimum number of reports that must be aggregated before a batch can be
+  collected.
 
 Public share:
-: The output of the VDAF sharding algorithm broadcast to each of the
+: The output of the VDAF sharding algorithm transmitted to each of the
   Aggregators. As defined in {{!VDAF}}.
 
 Report:
 : A cryptographically protected measurement uploaded to the Leader by a Client.
   Comprised of a set of report shares.
```

Comment:
```suggestion
  Includes a pair of report shares, one for each Aggregator.
```

(the switch from "comprise" to "include" is not for the normal pedantic reason, but because a report contains a few additional pieces of information, e.g. a report ID, client timestamp, public share. We could mention them all but I think that's too much detail, better to just shift language to note that the pair of report shares is only part of a Report.)